### ISSUE TITLE
make the launcher icon square

### ISSUE THREAD

**Issue by rocketinventor** [btd1337/Stupendously-Hot-Charmander#1]
In the Charmander Concept, the launcher icon is a square with edge to edge borders. This would mean that the [launcher_bfb.svg](https://github.com/btd1337/Stupendously-Hot-Charmander/blob/master/icons/launcher_bfb.svg) icon would have to be converted from a circle to a square (easy to do with any SVG editor,) and it's orange borders would need to be extended.

**Comment by btd1337** [comment]
In original concept the launcher icons is a circle, look this image.

![original concept 11](https://cloud.githubusercontent.com/assets/11943860/16200833/44b3e424-36e5-11e6-90f2-14fe4592a9a0.jpg)

When it is clicked, forms the orange square. Look:

![original concept 12](https://cloud.githubusercontent.com/assets/11943860/16200860/6d6ffc86-36e5-11e6-9dbb-e856915ed44e.jpg)

**Comment by rakshith-ravi** [comment]
I'd start working on it. Would changing the svg file do it all?

**Comment by btd1337** [comment]
This change affects more than svg file.

**Comment by rocketinventor** [comment]
@rakshithravi1997 If you look carefully at the launcher, you'll see that all icons have padding that's why it's more than svg.
@btd1337 This would be CSS thing, right?
I know CSS but, I've never done a Ubuntu theme before. What needs to be done?
unity.css?

**Comment by btd1337** [comment]
Read this: 
https://wiki.ubuntu.com/Unity/Theming
http://www.linux.org/threads/installing-obtaining-and-making-gtk-themes.8463/
http://worldofgnome.org/how-to-hack-gnomes-adwaita-gtk-theme/

**Comment by rakshith-ravi** [comment]
@rocketinventor Agreed. Just tried changing the svg file and it does seem to have a margin. Not sure where the change the margin though. What if we were to give the Unity bar itself a background of the same colour just for the first icon? Wouldn't that solve the issue and make it look like there's no padding? Not sure if it's possible though
@btd1337 This is not an enhancement. This is a necessity for the theme if we are making it as it is. Please remove the enhancement label

**Comment by rocketinventor** [comment]
@rakshithravi1997 If it's a theme, pretty much anything that is not a bug is an enhancement, right?
@btd1337 I read through the articles and did some more research on it. In Unity 7+, only `THEME_PATH/unity/launcher_bfb.{svg, png}` can be customized. :frowning: 
It seems like the launcher source itself would have to be modified.

**Comment by rakshith-ravi** [comment]
@rocketinventor Agreed. But I just didn't want every issue to have an enhancement tag :P
Let's just keep it for issues that might be good but isn't implemented in the original mock-up project. 

I am a programmer myself. If you can provide me with links on how to modify the source code for the unity launcher itself, I'd be happy to implement it

**Comment by btd1337** [comment]
I'm also learning about it...
